src/index.js

```diff
@@ -500,6 +500,31 @@ class PhotoshopAPI {
       throwError(err)
     }
   }
+
+  /**
+   * Play Photoshop Actions and then generate renditions and/or save a new psd
+   *
+   * @param {Input} input An object describing an input image file. Current support is for files less than 1000MB.
+   * @param {string|Output|Output[]} outputs Desired output
+   * @param {PhotoshopActionsOptions} options Photoshop Actions options
+   * @returns {Job} Photoshop Actions job
+   */
+  async photoshopActions (input, outputs, options) {
```

Comment:
I added an action before the other methods, maybe call this `applyPhotoshopActions`?

Reply:
Sure, I changed it to `applyPhotoshopActions`.